src/order/filter/lift.lean

```diff
@@ -336,15 +343,25 @@ le_antisymm
 
 lemma lift'_infi {f : ι → filter α} {g : set α → set β}
   [nonempty ι] (hg : ∀{s t}, g s ∩ g t = g (s ∩ t)) : (infi f).lift' g = (⨅i, (f i).lift' g) :=
-lift_infi $ by simp only [principal_eq_iff_eq, inf_principal, function.comp_app]; apply assume s t, hg
+lift_infi $ λ s t, by simp only [principal_eq_iff_eq, inf_principal, (∘), hg]
+
+lemma inf_lift' (f g : filter α) {s : set α → set β} (hs : ∀ {t₁ t₂}, s t₁ ∩ s t₂ = s (t₁ ∩ t₂)) :
```

Comment:
I think it should be named `lift'_inf` for consistency with `lift'_infi`, `lift_inf`.

Reply:
Should I rename this or `lift'_infi`? `lift'` is a `protected def` so `inf`/`infi` comes first in the formula.

Reply:
Even though `lift'` appears on the right for technical reasons, it's still "the `lift'` of the `inf`" on the left-hand side (and not "the `inf` of the `lift'`s"), so I think `lift'_inf` is a better name. I know this is technically not consistent with naming `(a + b) - c` as `add_sub`, but I think that can be justified because we read `+` and `-` as infix operations: "`a` plus `b` minus `c`".